lib/src/countly_flutter.dart

```diff
@@ -503,7 +503,6 @@ class Countly {
     log('Calling "onNotification"');
     await _channel.invokeMethod('registerForNotification').then((value) {
       callback(value.toString());
-      onNotification(callback);
```

Comment:
lets talk about this

Reply:
also test this peter with multiople push notifications and such https://countly.getoutline.com/doc/push-notifications-9qHAFIWhlc

Reply:
@peterBrxwn did you?